WMGM

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env python
 
+##############################################################
+#
+# This script is initiated by a daemon that communicates with the data submission on NiftyWeb.
```

Comment:
it says how it is initiated, but it doesn't say what it does (most important info)